packages/odyssey-react/src/components/Text/Text.theme.ts

```diff
@@ -24,9 +24,16 @@ export const theme: ThemeReducer = (theme) => ({
   /* eslint-enable @okta/odyssey/no-invalid-theme-properties */
   FontFamily: theme.FontFamilyBase,
 
-  LineHeight: theme.FontLineHeightBase,
-  NormalLineHeight: theme.FontLineHeightBase,
-  HeadingLineHeight: theme.FontLineHeightHeading,
+  LineHeight: theme.FontLineHeightBody,
+  BodyLineHeight: theme.FontLineHeightBody,
+  /* eslint-disable  @okta/odyssey/no-invalid-theme-properties */
+  Heading1LineHeight: theme.FontLineHeightHeading1,
+  Heading2LineHeight: theme.FontLineHeightHeading2,
+  Heading3LineHeight: theme.FontLineHeightHeading3,
+  Heading4LineHeight: theme.FontLineHeightHeading4,
+  Heading5LineHeight: theme.FontLineHeightHeading5,
+  Heading6LineHeight: theme.FontLineHeightHeading6,
+  /* eslint-enable  @okta/odyssey/no-invalid-theme-properties */
```

Comment:
bump ! can we update the lint rule please to allow numbers? We still have lots of t-shirt style spacing in our component prop APIs so we'll want to eventually unify things so developers can build a familiarity with how we present interfaces. Any reason for moving away from t-shirt size here? cc @josesolano-okta

Reply:
I wasn't aware of line heights being tokenized. But when it come to spacing we are moving away from t-shirt size because we want to introduce a larger set of space units and don't wan't to end up with xxxxxl down the line.

Reply:
Additional context, @jeffbelser-okta: each Heading text style now has a specific `line-height` that we're setting manually. These new tokens map directly to those styles, but don't exist on an ordinal scale.

Reply:
Lint rule updated!